docs/triggertemplates.md

```diff
@@ -15,6 +16,9 @@ spec:
       default: master
     - name: gitrepositoryurl
       description: The git repository url
+    - name: message
+      description: The message to print
+      default: This is the default message
```

Comment:
Maybe we want to remove default values here too?

Reply:
I think that you raise an important point to consider. It might not make sense to have a `default` value for Triggers parameters. However, I think that we should raise a separate issue to explore this idea. Under this issue, we might also want to consider creating our own Triggers Parameter types 👍